PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+# What you've done
+...
+
+# Why have you done it
+...
+
+# Screenshot (if visual)
+...
+
+# Any testing carried out
+...
```

Comment:
@chamilad the automatic resolvers are available regardless of the template. Also adding a very complicated PR template wouldn't make much sense in this case. Main idea is to provide a guide line to add a meaningful description so everyone has an idea what it is about.